### ISSUE TITLE
Broken remediations

### ISSUE THREAD

**Issue by ybznek** [OpenSCAP/scap-security-guide#1479]
```
$ make -C ./JRE/ clean; make -C ./JRE
```

File
**JRE/output/ssg-jre-ds.xml** : fix with id="java_jre_deployment_config_exists
```
<fix system="urn:xccdf:fix:script:sh" id="java_jre_deployment_config_exists" platform="Java Runtime Environment">

var_accounts_max_concurrent_login_sessions="
var_password_pam_unix_remember="
var_accounts_passwords_pam_faillock_deny="
var_accounts_passwords_pam_faillock_fail_interval="
var_accounts_passwords_pam_faillock_unlock_time="
var_accounts_user_umask="
var_accounts_user_umask="
var_auditd_action_mail_acct="
var_auditd_flush="
var_auditd_max_log_file="
var_auditd_max_log_file_action="
var_auditd_num_logs="
var_auditd_space_left_action="
var_selinux_policy_name="
var_selinux_state="
sshd_idle_timeout_value="
sshd_listening_port="<sub idref="xccdf_org.ssgproject.content_value_function_package_command" use="legacy"/>
package_command install aide
# Include source function library.
```
Fix seems to be quite broken.

When I revert ff8ae92cecbfa9b1fdc28f58676a06034f67a90b

The `<fix>` seems to be more "normal".
```
        <fix system="urn:xccdf:fix:script:sh" id="java_jre_deployment_config_exists">JAVA_CONFIG="/etc/.java/deployment/deployment.config"
JAVA_DIR="/etc/.java/deployment"

if [ ! -d ${JAVA_DIR} ] ; then
  mkdir -p -m 755 ${JAVA_DIR}
fi

if [ ! -e ${JAVA_CONFIG} ]; then
  touch ${JAVA_CONFIG} &amp;&amp; chmod 644 ${JAVA_CONFIG}
fi

chmod 644 ${JAVA_CONFIG}</fix>
```

I can see similar artifacts in other (not all) products.

**Comment by mpreisler** [comment]
CC @redhatrises

**Comment by redhatrises** [comment]
@ybznek you are right. This is definitely a blocker. I will take a look at this.

**Comment by redhatrises** [comment]
Fixed with #1481
